servicetalk-concurrent-test-internal/src/main/java/io/servicetalk/concurrent/test/internal/AwaitUtils.java

```diff
@@ -30,10 +30,22 @@ private AwaitUtils() {
         // no instances
     }
 
+    /**
+     * Await a {@link CountDownLatch} until the latch is counted down.
+     * @param latch the {@link CountDownLatch} to await.
+     * @throws InterruptedException if the current thread is interrupted while waiting or has its interrupted status set on
+     * entry to this method
+     */
+    public static void await(CountDownLatch latch) throws InterruptedException {
+        latch.await();
```

Comment:
I see the pain with adding `throws Exception` to all tests that don't have it. Thanks for highlighting it and opening a PR before you finished to change all files.

Instead of changing gazillion test files, we can re-throw it bypassing compiler checks for checked exceptions:
```java
    public static void await(CountDownLatch latch) {
        try {
            latch.await();
        } catch (InterruptedException e) {
            Thread.currentThread().interrupt();
            throwException(e);
        }
```
Where `throwException` is statically imported from `io.servicetalk.utils.internal.PlatformDependent` (you will need to add an `implementation` dependency for `servicetalk-utils-internal`).
This is a dirty hack, but if you can verify this approach work for jUnit5, I'm fine with it. We can improve it over time. More important is to make sure jUnit5 can terminate tests when they exceed timeout. 

To verify, try to write a test that does something that never completes:
```java
AwaitUtils.await(new CountDownLatch(1));
```
and make sure jUnit5 fails this test with a timeout exception.